diff --git a/GNUmakefile.in b/GNUmakefile.in
index 030a3f39..318866c4 100644
--- a/GNUmakefile.in
+++ b/GNUmakefile.in
@@ -64,6 +64,9 @@ CFLAGS := @CFLAGS@
 LDFLAGS := @LDFLAGS@
 LIBS := @LIBS@
 
+MFORMAT ?= mformat
+MCOPY ?= mcopy
+
 CC_FOR_TARGET := @CC_FOR_TARGET@
 export CC_FOR_TARGET
 LD_FOR_TARGET := @LD_FOR_TARGET@
@@ -222,8 +225,8 @@ ifneq ($(BUILD_UEFI_CD),no)
 		LIMINE_UEFI_CD_SIZE="$$(find "$$LIMINE_UEFI_CD_TMP" -type f -exec cat '{}' + | wc -c)" && \
 		LIMINE_UEFI_CD_SECTORS="$$(( (($$LIMINE_UEFI_CD_SIZE + 1048575) / 1048576 + 1) * 2048 ))" && \
 		dd if=/dev/zero of='$(call SHESCAPE,$(BINDIR))/limine-uefi-cd.bin' bs=512 count="$$LIMINE_UEFI_CD_SECTORS" 2>/dev/null && \
-		mformat -i '$(call SHESCAPE,$(BINDIR))/limine-uefi-cd.bin' -T "$$LIMINE_UEFI_CD_SECTORS" -N 12345678 :: && \
-		mcopy -D o -s -m -i '$(call SHESCAPE,$(BINDIR))/limine-uefi-cd.bin' "$$LIMINE_UEFI_CD_TMP"/EFI :: && \
+		$(MFORMAT) -i '$(call SHESCAPE,$(BINDIR))/limine-uefi-cd.bin' -T "$$LIMINE_UEFI_CD_SECTORS" -N 12345678 :: && \
+		$(MCOPY) -D o -s -m -i '$(call SHESCAPE,$(BINDIR))/limine-uefi-cd.bin' "$$LIMINE_UEFI_CD_TMP"/EFI :: && \
 		rm -rf "$$LIMINE_UEFI_CD_TMP"
 endif
 
