--- make-4.4.1-clean/src/main.c
+++ make-4.4.1/src/main.c
@@ -1430,6 +1430,8 @@
 
   /* Initialize the special variables.  */
   define_variable_cname (".VARIABLES", "", o_default, 0)->special = 1;
+  /* Initialize the INREPO variable for Znu */
+  define_variable_cname ("INREPO", "yes", o_default, 0);
   /* define_variable_cname (".TARGETS", "", o_default, 0)->special = 1; */
   define_variable_cname (".RECIPEPREFIX", "", o_default, 0)->special = 1;
   define_variable_cname (".SHELLFLAGS", "-c", o_default, 0);
